cmd/sink/logs/logs.go

```diff
@@ -0,0 +1,81 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package logs
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/kubearchive/kubearchive/cmd/sink/k8s"
+	ocel "github.com/kubearchive/kubearchive/pkg/cel"
+	"github.com/kubearchive/kubearchive/pkg/log_urls"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	configCmName     = "kubearchive-config"
+	loggingCmKeyName = "logging-configmap-name"
```

Comment:
We can do that and it will make it so that we don't have to change the code here if the the name of the `ConfigMap` or the key name for the logging `ConfigMap` change. However, there will still need to be a consts here with the names of the environment variables

Reply:
unless you mean something different in which case, can you show me an example of what you're talking about?

Reply:
Something simple like this:
```spec:
  mariaDbRef:
    name: kubarchive
```

Reply:
done!